### PR TITLE
docs: Recommend node18

### PR DESCRIPTION
**Pull request by SeaDve**
Also, remove unneeded yarn installation as it is included in the SDK

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -21,7 +21,6 @@ export FLATPAK_ENABLE_SDK_EXT=node14
 
 5. Within the integrated terminal of your editor, execute the following commands at the root of the repository:
 ```bash
-npm install --global yarn
 yarn install
```
Comment (Hofer-Julian) [inline CONTRIBUTING.md:24]: I guess we can just recommend npm nowadays?

**Review by Hofer-Julian — APPROVED** [review]
Looks good to me

File: CONTRIBUTING.md (hunk 1 above)
Comment (bilelmoussaoui) [reply CONTRIBUTING.md:24]: /me still prefers yarn over npm

Comment (SeaDve) [reply CONTRIBUTING.md:24]: It seems to be installed with the [SDK](https://github.com/flathub/org.freedesktop.Sdk.Extension.node18/blob/cfdcf070731c5b1914af08ad1b97564e5a8daa97/org.freedesktop.Sdk.Extension.node18.yaml#L41)

Comment (bilelmoussaoui) [reply CONTRIBUTING.md:24]: > I trust you guys with the recommendations, but don’t you have to install yarn first? Or is it nowadays packaged with npm?

this is the instructions for developing with a flatpaked vscode

**Comment by bilelmoussaoui** [comment]
if we are going to recommend node18, we need to add that to our CI as well.

**Comment by SeaDve** [comment]
I think we can drop 12.x too?

File: .github/workflows/CI.yml [hunk 2]
```diff
@@ -21,7 +21,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [12.x, 14.x]
+        node-version: [14.x, 18.x]
```
Comment (bilelmoussaoui) [inline .github/workflows/CI.yml:24]: could you add 16.x as well?

**Pull request merged**